leapp/cli/upgrade/__init__.py

```diff
@@ -14,6 +17,46 @@
 from leapp.utils.output import report_errors, beautify_actor_exception
 
 
+def logs_handler():
+    """ Creates log structure and archive log files from a previous run of Leapp """
+    archive_dir = '/var/log/leapp/archive/'
+    logs_dir = '/var/log/leapp'
+    debug_dir = '/var/log/leapp/dnf-debugdata/'
+    leapp_db = '/var/lib/leapp/leapp.db'
+    log_files = [
+        '/var/log/leapp/leapp-upgrade.log',
+        '/var/log/leapp/leapp-report.txt',
+        '/var/log/leapp/dnf-plugin-data.txt'
+    ]
+    files_to_archive = [file for file in log_files if os.path.isfile(file)]
+
+    if os.path.isdir(debug_dir) and os.listdir(debug_dir):
+        files_to_archive.append(debug_dir)
+
+    if not os.path.isdir(archive_dir):
+        os.makedirs(archive_dir)
+
+    if files_to_archive:
+        archive_file = os.path.join(archive_dir, 'leapp-{}-logs.tar.gz'.format(int(time.time())))
+
+        with tarfile.open(archive_file, "w:gz") as tar:
+            for file in files_to_archive:
+                tar.add(file)
+                if os.path.isdir(file):
+                    shutil.rmtree(file, ignore_errors=True)
+                try:
+                    os.remove(file)
+                except OSError:
+                    pass
+            if os.path.isfile(leapp_db):
+                tar.add(leapp_db)
+
+    if os.getenv('LEAPP_DEBUG', '0') == '1':
+        logs_dir = os.path.join(logs_dir, 'dnf-debugdata')
+    if not os.path.isdir(logs_dir):
+        os.makedirs(logs_dir)
```

Comment:
Do not create the dir here, but instead put it to spec file:
[line 186](https://github.com/oamg/leapp/blob/master/packaging/leapp.spec#L186): `install -m 0755 -d %{buildroot}%{_localstatedir}/log/leapp`
[line 220](https://github.com/oamg/leapp/blob/master/packaging/leapp.spec#L220): `%dir %{_localstatedir}/log/leapp`